yb-voyager/cmd/exportData.go

```diff
@@ -576,7 +583,12 @@ func addLeafPartitionsInTableList(tableList []sqlname.NameTuple, ifTableListSet
 func GetRootTableOfPartition(table sqlname.NameTuple) (sqlname.NameTuple, error) {
 	parentTable := source.DB().ParentTableOfPartition(table)
 	if parentTable == "" {
-		return table, nil
+		//now we know its a root table so return the tuple from the nameregistry
```

Comment:
can you give more context in the comment on why you need to lookup from nameregistry again? 
For my clarification: this is because the original tuples do not have the target-side of names, right?

Reply:
Yes, right. The original tuples are hand-crafted ones without target names, so for the root, we shouldn't use these but instead get a proper one from the name registry.